pytradfri/TODO

```diff
@@ -0,0 +1,35 @@
+--> Commit 2 --
```

Comment:
This file should not be included I think.

Reply:
ups.